models/classes/LtiRoles.php

```diff
@@ -26,20 +26,24 @@
  */
 interface LtiRoles
 {
-    const CLASS_URI = 'http://www.tao.lu/Ontologies/TAOLTI.rdf#LTIRole';
+    public const CLASS_URI = 'http://www.tao.lu/Ontologies/TAOLTI.rdf#LTIRole';
 
-    const PROPERTY_URN = 'http://www.tao.lu/Ontologies/TAOLTI.rdf#RoleURN';
+    public const PROPERTY_URN = 'http://www.tao.lu/Ontologies/TAOLTI.rdf#RoleURN';
 
-    const INSTANCE_LTI_BASE = 'http://www.tao.lu/Ontologies/TAOLTI.rdf#LtiBaseRole';
+    public const INSTANCE_LTI_BASE = 'http://www.tao.lu/Ontologies/TAOLTI.rdf#LtiBaseRole';
 
-    const CONTEXT_TEACHING_ASSISTANT = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#TeachingAssistant';
+    public const CONTEXT_TEACHING_ASSISTANT = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#TeachingAssistant';
 
-    const CONTEXT_LEARNER = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#Learner';
+    public const CONTEXT_LEARNER = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#Learner';
 
-    const CONTEXT_INSTRUCTOR = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#Instructor';
+    public const CONTEXT_INSTRUCTOR = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#Instructor';
 
-    const CONTEXT_ADMINISTRATOR = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#Administrator';
+    public const CONTEXT_ADMINISTRATOR = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#Administrator';
 
     // LTI 1p3
-    const CONTEXT_LTI1P3_LEARNER = 'http://purl.imsglobal.org/vocab/lis/v2/membership#Learner';
+    public const CONTEXT_LTI1P3_LEARNER = 'http://purl.imsglobal.org/vocab/lis/v2/membership#Learner';
+
+    public const CONTEXT_LTI1P3_MENTOR = 'http://purl.imsglobal.org/vocab/lis/v2/membership#Mentor';
+
+    public const CONTEXT_LTI1P3_INSTRUCTOR = 'http://purl.imsglobal.org/vocab/lis/v2/membership#Instructor';
```

Comment:
Let's add the remaining ones as well.
- `http://purl.imsglobal.org/vocab/lis/v2/membership#Administrator`
- `http://purl.imsglobal.org/vocab/lis/v2/membership#ContentDeveloper`
- `http://purl.imsglobal.org/vocab/lis/v2/membership#Manager`
- `http://purl.imsglobal.org/vocab/lis/v2/membership#Member`
- `http://purl.imsglobal.org/vocab/lis/v2/membership#Officer`

To be fair, there's actually more, as each role has its sub-roles, e.g. `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#Reviewer`. Perhaps, it's worth adding them too.

- `http://purl.imsglobal.org/vocab/lis/v2/membership/Administrator#Administrator`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Administrator#Developer`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Administrator#ExternalDeveloper`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Administrator#ExternalSupport`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Administrator#ExternalSystemAdministrator`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Administrator#Support`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Administrator#SystemAdministrator`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/ContentDeveloper#ContentDeveloper`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/ContentDeveloper#ContentExpert`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/ContentDeveloper#ExternalContentExpert`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/ContentDeveloper#Librarian`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#ExternalInstructor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#Grader`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#GuestInstructor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#Lecturer`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#PrimaryInstructor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#SecondaryInstructor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#TeachingAssistant`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#TeachingAssistantGroup`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#TeachingAssistantOffering`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#TeachingAssistantSection`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#TeachingAssistantSectionAssociation`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#TeachingAssistantTemplate`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Learner#ExternalLearner`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Learner#GuestLearner`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Learner#Instructor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Learner#Learner`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Learner#NonCreditLearner`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Manager#AreaManager`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Manager#CourseCoordinator`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Manager#ExternalObserver`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Manager#Manager`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Manager#Observer`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Member#Member`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#Advisor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#Auditor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#ExternalAdvisor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#ExternalAuditor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#ExternalLearningFacilitator`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#ExternalMentor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#ExternalReviewer`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#ExternalTutor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#LearningFacilitator`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#Mentor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#Reviewer`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor#Tutor`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Officer#Chair`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Officer#Communications`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Officer#Secretary`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Officer#Treasurer`
- `http://purl.imsglobal.org/vocab/lis/v2/membership/Officer#Vice-Chair`

Reply:
good point. will add for the future